### DISCUSSION TITLE
NMR templates and protocols

### DISCUSSION THREAD

**Discussion by NRayya** [NFDI4Chem/nmrxiv#651] [General]
Varying uses of NMR in the chemical domains and fields require varying preparation and analyzing metadata that we plan to capture in nmrXiv by using templates corresponding to those fields. Each template will have the needed fields for metadata along with protocols describing each step of the workflow. For example, a sample taken from soil will be handled differently from a sample taken from an animal. 

Candidate fields/ sample types

- "Pure"/commercial sample
- synthesized compounds 
- metabolomics
  - one organism
  - one organism hosting another one
  - biofilm
  - other?
- environmental sample

In addition to the fields themselves, their naming is open to discussion.

**Comment by steinbeck** [comment]
I think the vast majority of samples measured at academic labs will be from synthesized compounds, so I would do them first.

**Comment by NRayya** [comment]
To be closed in favor of the available NMR MIChI now
